test/integration/verify.test.js

```diff
@@ -127,6 +138,60 @@ describe('verify batch content', () => {
     expect(files.filter(x => x === PROCESSED_CTL_CHECKSUM_BLOB_NAME).length).toBe(1)
   })
 
+  test('renames glos batch file on success if all files present and hash valid', async () => {
```

Comment:
The other test names have 'Glos' rather than 'glos', so was thinking change this for consistency